src/components/ProjectListItemCard/ProjectListItemCard.tsx

```diff
@@ -53,28 +53,16 @@ const useStyles = makeStyles(({ spacing }) => ({
   },
 }));
 
-export interface ProjectListItemCardProps {
-  id: string;
-  projectImagePath: string;
-  name: string;
-  countryName: string;
-  region: string;
-  sensitivity: SensitivityType;
-  status: ProjectStatus;
-  numberOfLanguageEngagements: number;
-  esadDate: string;
-}
+export type ProjectListItemCardProps = Partial<Project> & {
+  projectImagePath?: string; // where is this comming?
+  numberOfLanguageEngagements?: number; // where is this comming?
```

Comment:
Ok so `projects()` returns a list of `Project`s, which is an interface. There are two concrete types implementing this interface, `InternshipProject` and `TranslationProject`. Both of these concrete types have an `engagements` property, which is a list that has a total count on it.  We can use this syntax below to access their properties.

```graphql
items {
  ... on InternshipProject {
    engagements {
      total
    }
  }
  ... on TranslationProject {
    engagements {
      total
    }
  }
}
```
That ^ can be put on the Project fragment mentioned in my other comment along with the other properties that are shared on the Project interface.